Files/View Models/Properties/FolderProperties.cs

```diff
@@ -80,7 +80,17 @@ public async override void GetSpecialProperties()
             }
 
             StorageFolder storageFolder;
-            var isItemSelected = await CoreApplication.MainView.ExecuteOnUIThreadAsync(() => App.CurrentInstance.ContentPage.IsItemSelected);
+            bool isItemSelected;
+
+            try
+            {
+                isItemSelected = await CoreApplication.MainView.ExecuteOnUIThreadAsync(() => App.CurrentInstance.ContentPage.IsItemSelected);
+            }
+            catch
```

Comment:
@Thomas1664 I'm curious, what kind of exception is thrown here?

Reply:
@gave92 It throws a `System.NullReferenceException` because `ContentPage` is `null` if you are on `Home`.